README.md

```diff
@@ -43,7 +45,9 @@ OR
 
 Include **'therubyracer'** in your Gemfile.
 
-    gem "therubyracer", :require => 'v8'
+```bash
```

Comment:
Is `bash` the right syntax highlighting choice here? Might need `ruby` or something.